controlpanel/frontend/jinja2/help.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% set page_title = "Help" %}
+
+{% block content %}
+  <div class="govuk-grid-row" id="reset-container">
+      <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Help</h1>
+
+    <p class="govuk-body">Common queries, support tasks and problems are
+    explained in the following videos.</p>
+
+    <p class="govuk-body">Should you still require help, please visit our
```

Comment:
Question: Should we add a paragraph or mention the User Guidance here as well so that we somehow nudge users to look at the documentation before raising a support request? Thoughts?

Reply:
Yes we should. Feedback and suggestions like this are exactly what I'm looking for. While the code is in place, the copy for what should be on the page is not... so feel free to tell me what you think should be there. :+1: